code/hud/hud.cpp

```diff
@@ -3744,7 +3744,7 @@ int hud_objective_notify_active()
  * @param wiggedy_wack
  * @param eye_orient 
  */
-void HUD_set_offsets(object *viewer_obj, int wiggedy_wack, matrix *eye_orient)
+void HUD_set_offsets()
 {
 	if ( Viewer_mode & ( VM_TOPDOWN | VM_CHASE ) ) {
```

Comment:
It looks like the logic from the old function call
```
int dont_offset = ((Game_mode & GM_MULTIPLAYER) && (Net_player->flags & NETINFO_FLAG_OBSERVER));
```
should be rolled into this condition.

Reply:
That condition was never actually applied to nose, only to offset.
And just because you're an observer, doesn't mean that your hud can now suddenly move massively different IMO

Reply:
Ok, I suppose not.